alacritty/src/input.rs

```diff
@@ -394,17 +394,22 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
                 ClickState::TripleClick
             }
             _ => {
-                // Don't launch URLs if this click cleared the selection
-                self.ctx.mouse_mut().block_url_launcher = !self.ctx.selection_is_empty();
+                // Only clear selection if left or middle button are pressed
+                if mouse.left_button_state == ElementState::Pressed
+                    || mouse.middle_button_state == ElementState::Pressed
```

Comment:
Why would we remove it for RMB, but still allow selecting with MMB? To me it seems like we'd either use LMB only, or all of them?